runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesConfig.java

```diff
@@ -34,9 +34,9 @@
 public interface InterfaceStylesConfig extends Config
 {
 	@ConfigItem(
-		keyName = "gameframe",
-		name = "Gameframe",
-		description = "The gameframe to use for the interface"
+			keyName = "gameframe",
```

Comment:
Revert this indentation change

```suggestion
		keyName = "gameframe",
```